Forte.React.AspNetCore/JsonSerializationService.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.IO;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+using Jering.Javascript.NodeJS;
+
+namespace Forte.React.AspNetCore;
+
+internal class JsonSerializationService : IJsonService
+{
+    private readonly JsonSerializerOptions _jsonSerializerOptions;
+
+    public JsonSerializationService(JsonSerializerOptions jsonSerializerOptions)
+    {
+        _jsonSerializerOptions = jsonSerializerOptions;
+    }
+
+    public ValueTask<T?> DeserializeAsync<T>(Stream stream,
+        CancellationToken cancellationToken = new CancellationToken())
```

Comment:
as cancellation token is not used please consider using CancellationToken.None

Reply:
Fixed